ustriage/ustriage.py

```diff
@@ -193,7 +193,7 @@ def reverse_auto_date_range(start, end):
         return "Monday triage"
 
     if start == end:
-        if start_weekday in [5, 6]:
+        if start_weekday in [4, 5, 6]:
             return None  # weekend: process not specified
```

Comment:
I agree that running it on Saturday will pick Friday (= day 4) and `start==end` will be true.
So adding the "4" here seems right.

I was thinking if the 6 has to be dropped. But if you really run it on a Monday it will not be `start==end`.
And if someone manually specifies "just Sunday" then it is fine to not have a name for this triage.
So it is ok to keep the 6 as well.

Reply:
Whilst I think this is unrelated to here (or is it not?), I somewhat understand what you're saying but I think it's "safer" to keep it if it doesn't do any harm? I mean, I guess it's certainly not wrong to have 6 in the list so I think it's OK to keep it unless we hit some sorta bug that would question its very existence. What do you think?

Reply:
Yeah as I said - it is ok to keep the 6 in there.
This comment was more about documenting my thought process in case anyone feels differently on this.